doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2941,9 +2941,13 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       .. exn:: Too few occurrences.
          :undocumented:
 
-   .. tacv:: change @term {? {? at {+ @natural}} with @term} in @ident
+   .. tacv:: change @term {? {? at {+ @natural}} with @term} in @goal_occurrences
```

Comment:
There's no nonterminal names `goal_occurrences`, despite being used in many places in this chapter, which has not been updated.  The current syntax shown here is pretty incorrect.  It's something like this (below) in `fullGrammar`, before editing by doc_grammar.  Also, doc_grammar seems to be incorrectly dropping `conversion` somehow, which I'll need to investigate.

Maybe it's best to wait on updating this part of the doc?

```
simple_tactic: [
| "change" conversion clause_dft_concl

conversion: [
| constr
| constr "with" constr
| constr "at" occs_nums "with" constr

clause_dft_concl: [
| "in" G_TACTIC_in_clause
| occs

G_TACTIC_in_clause: [
| "*" occs
| "*" "|-" concl_occ
| LIST0 hypident_occ SEP "," "|-" concl_occ
| LIST0 hypident_occ SEP ","